test/scheduler_visualisation_test.py

```diff
@@ -585,7 +638,11 @@ class X(luigi.Task):
 
     def test_worker_list_disabled_worker(self):
         class X(luigi.Task):
-            pass
+            def run(self):
```

Comment:
Can you use the RunOnceTask instead of doing this charade everywhere? https://github.com/spotify/luigi/search?q=RunOnceTask&unscoped_q=RunOnceTask

Reply:
Why i didn't notice this class before =\